.github/workflows/pr-percy-snapshots.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Remove SCM directories that will be replaced by artifact files
-        run: rm -rf templates/docs/examples/ templates/_macros/ tokens/ sd.config.json scss/
+        run: rm -rf templates/docs/examples/ templates/_macros/ tokens/ sd.config.mjs scss/
```

Comment:
These Percy file changes will have to be merged to `main` separately before this can be tested